optimade/server/tests/test_more_structures.json

```diff
@@ -1,30 +1,30 @@
-{
-  "id": "db/1234567",
+[{
+  "task_id": "db/1234567",
   "type": "structure",
 	"last_modified": {
 		"$date": "2019-06-08T05:13:37.331Z"
 	},
-  "_exmpl_band_gap": 1.23456,
+  "band_gap": 1.23456,
+	"_mp_chemsys": "Ge-Si",
   "elements": ["Si", "Ge"],
-  "nelements": 3,
+  "nelements": 2,
+  "elements_ratios": [0.5, 0.5],
+  "pretty_formula": "SiGe",
+  "formula_anonymous": "GeSi",
```

Comment:
As stated above, I don't think the test cases should include hardcoded aliased fields like this

Reply:
What in particular do you mean here?
For me, this particular file is only to test that a valid result can be returned during testing.
I regularly change different keys and stuff to test if aliasing, error-reporting, etc, work as intended.

But I guess, the best way would be to create a new structure each time, that displays each of the different properties I am currently investigating and create actual separate tests for these.... buuut :sweat_smile: